.github/workflows/publish-release.yml

```diff
@@ -23,7 +33,25 @@ jobs:
           packageUser: ${{ github.actor }}
           packagePAT: ${{ secrets.BALLERINA_STDLIB_TOKEN }}
         run: |
-          NEW_VERSION=$(echo "${GITHUB_REF}" | cut -d "/" -f3)
-          echo "New version: ${NEW_VERSION}"
-          echo "Github username: ${GITHUB_ACTOR}"
-          ./gradlew -PpublishToCentral=true -Pversion=${NEW_VERSION} publish
+          ./gradlew release -Prelease.useAutomaticVersion=true
+      - name: Create release
+        run: |
+          curl --request POST 'https://api.github.com/repos/ballerina-platform/module-ballerina-sql/releases' \
+          --header 'Accept: application/vnd.github.v3+json' \
+          --header 'Authorization: Bearer ${{ secrets.BALLERINA_STDLIB_TOKEN }}' \
```

Comment:
Are we going to use the same token OR a new one here? @daneshk @shafreenAnfar @ThisaruGuruge @ldclakmal